python/semantic_kernel/semantic_functions/prompt_template_with_data_config.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+
+from semantic_kernel.semantic_functions.prompt_template_config import (
+    PromptTemplateConfig,
+)
+
+
+@dataclass
+class PromptTemplateWithDataConfig(PromptTemplateConfig):
```

Comment:
Please move this class and file into the connectors/ai/open_ai folders since it is OpenAI specific.

Reply:
and maybe it should even be a subclass of OpenAIChatPromptTemplate, since it is a chat feature and then Azure specific.

Reply:
I don't think it can be a subclass of OpenAIChatPromptTemplate, since it's a PromptTemplateConfig and not a PromptTemplate

Reply:
Moved to the open_ai folders and renamed to OpenAIChatPromptTemplateWithDataConfig (nice and long name...). Still inherits from PromptTemplateConfig.